.gitignore

```
@@ -20,6 +21,7 @@ misc/*
 src/release.h
 appendonly.aof*
 appendonlydir
+appendonlydir-*
```

```suggestion
appendonlydir*
```
Would be more compact

---

Do you also want to make the others more compact? (dump*.rdb, nodes*.conf)
The reason I did this is that one explicitly captures default and the other explicitly captures create-cluster. And `*.log` already does it so i didn't change it.

---

I guess making the other ones more compact also makes sense. I think it makes the file a bit easier to read and easier to understand this isn't a special case, just that it supports more arguments.